modules/processor/header-enricher/config/header-enricher.groovy

```diff
@@ -26,7 +27,7 @@ beans {
 	
 	si.'header-enricher'('input-channel':'input','output-channel':'output') {
 		result.each {k,v->
-			si.'header'(name:k,expression:v)
+			si.'header'(name:k,expression:v,overwrite:overwrite)
```

Comment:
Having `overwrite` as a common option, I'd suggests to do like this:
```
si.'header-enricher'('input-channel':'input','output-channel':'output', 'default-overwrite':overwrite) 
```
Instead of overheading on each `header`